extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -210,4 +218,27 @@ interface HibernateOrmConfigDevUI {
         @WithDefault("false")
         boolean allowHql();
     }
+
+    @Deprecated(since = "3.24", forRemoval = true)
+    @ConfigGroup
+    interface HibernateOrmConfigMapping {
+
+        /**
+         * Mapping format.
+         */
+        HibernateOrmConfigMappingFormat format();
+
+        @Deprecated(since = "3.24", forRemoval = true)
+        @ConfigGroup
+        interface HibernateOrmConfigMappingFormat {
+            /**
+             * How the default JSON/XML format mappers are configured.
+             *
+             * @deprecated Only available to mitigate migration from the Quarkus preconfigured current mappers
+             */
+            @Deprecated(since = "3.24", forRemoval = true)
```

Comment:
Doesn't usage of deprecated properties trigger warnings? If so I don't think we want to deprecate this just now... It's a way to get applications _more_ up-to-date.